### ISSUE TITLE
Zoom on a GIF/picture with mouse click

### ISSUE THREAD

**Issue by tahnik** [tahnik/devRantron#117]

**Comment by TheWeirdDev** [comment]
Why wontfix :( ?
It's really hard to read the code in some images, we have to drag & drop the image in a browser window to be able to see it in the real size.

**Comment by hex2f** [comment]
I have to agree on this. I think we should have some feature to show a bigger version of an image.

**Comment by LoganS1** [comment]
I also agree, here is an example I just saw:
![capture](https://user-images.githubusercontent.com/21230472/31857766-c5f2d8b4-b6ab-11e7-9b66-8b9069acd5c4.PNG)
